### PR TITLE
target HHVM 4.108, fix outdated content

### PR DESCRIPTION
**Pull request by jjergus**
(you can look at each commit separately for easier review)

See https://github.com/hhvm/hhvm.com/pull/125

### CODE REVIEW

File: src/codegen/PRODUCT_TAGS.php [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
+ * @generated SignedSource<<9273e4ec8cb5789cc665443973de0f65>>
  */
 namespace HHVM\UserDocumentation;
 
 const dict<APIProduct, string> PRODUCT_TAGS = dict[
-  APIProduct::HACK => 'HHVM-4.102.0',
-  APIProduct::HSL => 'v4.94.0',
-  APIProduct::HSL_EXPERIMENTAL => 'v4.99.0',
+  APIProduct::HACK => 'HHVM-4.108.0',
+  APIProduct::HSL => 'v4.108.1',
```
Comment (fredemmott) [inline src/codegen/PRODUCT_TAGS.php:11]: Are things like 'HH\Lib\Vec\map()` still listed under HSL API reference after this bump?

File: src/build/APISourcesBuildStep.php [hunk 2]
```diff
@@ -29,22 +29,13 @@ final class APISourcesBuildStep extends BuildStep {
...
       ],
     ),
-    APIProduct::HSL => shape(
-      'name' => 'hhvm/hsl',
-      'prefixes' => keyset['src/'],
-    ),
     APIProduct::HSL_EXPERIMENTAL => shape(
       'name' => 'hhvm/hsl-experimental',
       'prefixes' => keyset[
         'src/debug/',
```
Comment (fredemmott) [inline src/build/APISourcesBuildStep.php:38]: str/ should be added

File: src/build/HHAPIDocBuildStep.php [hunk 3]
```diff
@@ -74,7 +74,8 @@ public function buildAll(): void {
     // We have prefix whitelists (especially for hsl-experimental), but they're
     // already handled when extracting the tarball; no need to filter out non-
     // whitelisted files again here.
-    $hsl_sources = self::findSources(BuildPaths::HSL_TREE.'/src/', $exts);
+    $hsl_sources =
+      self::findSources(BuildPaths::HHVM_TREE.'/hphp/hsl/src/', $exts);
```
Comment (jjergus) [inline src/build/HHAPIDocBuildStep.php:78]: > Are things like 'HH\Lib\Vec\map()` still listed under HSL API reference after this bump?

Yes, thanks to this change.

**Review by fredemmott — APPROVED** [review]
Needs https://github.com/usox/hackttp/commit/a5ba0a2c4dc31f06b4907026b3bc0bd208a07a24 pulling in

**Pull request merged**